kvmagent/ansible/kvm.py

```
@@ -407,6 +495,8 @@ def rpm_based_install():
 
         #we should check libvirtd config file status before restart the service
         libvirtd_conf_status = update_libvirtd_config(host_post_info)
```

Ruff (0.15.5)

[error] Local variable `libvirtd_conf_status` is assigned to but never used

Remove assignment to unused variable `libvirtd_conf_status`

(F841)